tfx/extensions/google_cloud_ai_platform/training_clients.py

```diff
@@ -149,6 +149,35 @@ def get_job_name(self) -> Text:
     """Gets the job name."""
     return self._job_name
 
+  def generate_container_command(self,
```

Comment:
How about adding an underscore to mark this method  as a protected method like '_generate_container_command()' if this method will be called only inside the class (or subclasses)?

https://google.github.io/styleguide/pyguide.html#3162-naming-conventions

Reply:
So, the reason I made it public was so that inheriting classes outside of TFX can reuse it. The idea is that at Twitter we are building a similar Trainer using KF TFJob instead. With this method public, we can just inherit the client and generate the container command without having to vendor the code.

Given our class will be in an inherited class, then theoretically, you should not use protected or private methods. 

Also, making it public ensures a bit more of stability for us to derive from and the cost is low due to low complexity of underlying code.

Let me know if that's okay, otherwise I can make it private.